docs/quick-start-docker-cluster.md

```diff
@@ -0,0 +1,164 @@
+
+* [Docker](https://docs.docker.com/install/)
+* [docker-compose](https://docs.docker.com/compose) >= version 1.6
+* [Go](https://golang.org/doc/install) (needed to build metrictank)
+   * Ensure that `$GOPATH/bin` is added to your `$PATH`
+
+## Getting the repository
+
+Pull down the repository and its dependencies: 
+
+```
+go get github.com/grafana/metrictank/...
+cd $GOPATH/src/github.com/grafana/metrictank
+```
+
+## Build metrictank
+
+Build metrictank and the docker images:
+```
+make
+```
+
+`make` uses/creates the directory `$GOPATH/src/github.com/grafana/metrictank/build`. If you tried to spin up the stack prior to building metrictank then docker already created this directory, as root, and it will need to be deleted (then run `make` again).
+
+## Bring up the stack
+
+The stack will listen on the following ports:
+
+* 80    tcp (carbon)
+* 2003  tcp (metrictank's carbon input)
+* 2181  tcp (kafka)
+* 3000  tcp (grafana's http port)
+* 6060  tcp (metrictank's internal endpoint)
+* 6061  tcp (caddy)
+* 6062  tcp (caddy)
+* 6063  tcp (caddy)
+* 6831  udp (jaeger)
+* 8080  tcp (the graphite query endpoint)
+* 8081  tcp (caddy)
+* 8082  tcp (caddy)
+* 8125  udp (statsd endpoint)
+* 9042  tcp (cassandra)
+* 9090  tcp (prometheus)
+* 9092  tcp (kafka)
+* 9100  tcp (prometheus node exporter)
+* 9999  tcp (kafka)
+* 16686 tcp (jaeger)
+
+If you already have something else listening on those ports (such as a carbon or grafana server), shut it down, as it will conflict.
+
+
+Inside your copy of the repository, you can bring up the stack like so:
+
+```
+cd docker/docker-cluster
+docker-compose up
+```
+
+If this gives you the error `service 'version' doesn't have any configuration options`,
+your version of `docker-compose` is too old and you need to update to >=1.6.
+
+Wait until the stack is up.
+
+## Working with Grafana and metrictank
+
+In your browser, open Grafana at `http://localhost:3000` (or your docker-machine address) and log in as `admin:admin`.  
+If Grafana prompts you to change the password, you can skip it, since it doesn't matter for a local test setup.  
+
+### Sending and visualizing data
+
+In the "+" (Create) menu, hit `Dashboard`.  
+This opens the dashboard editor and has a selector open to add a new panel.  Hit "Graph" for graph panel.  
+The panel will appear but not contain data yet.  ([Grafana documentation for graph panel](http://docs.grafana.org/features/panels/graph/)).
+Click on the title of the panel and hit 'edit'.
+In the metrics tab you should see a bunch of metrics already in the root hierarchy:
+
+* `service_is_statsdaemon`: statsdaemon's own internal metrics which it sends to metrictank's carbon port.
+* `metrictank`: internal stats reported by metrictank
+* `stats`: metrics aggregated by statsdaemon and sent into metrictank every second. Will only show up if something actually sends
+  metrics into statsdaemon (e.g. if graphite receives requests directly, you send stats to statsdaemon, etc)
```

Comment:
this is true for docker-standard, but different for docker-cluster, as we use a separate monitoring server (graphite) to receive all this data. you'd only see this if you first select the monitoring datasource.

but i think everything after line 83 should just be scrapped out of this guide.
it's going to be too much to maintain all these details for little benefit.

users new to the project should just follow the normal quickstart guide, and if they succeeded at that, they can do this one, so we don't have to go into all this detail again.